README.rst

```diff
@@ -71,7 +71,8 @@ Please note that on Linux installing voice you must install the following packag
 * libffi-dev (or ``libffi-devel`` on some systems)
 * python-dev (e.g. ``python3.6-dev`` for Python 3.6)
 
-
+Examples
```

Comment:
I personally think this should be solved by changing the name of Quick Example as we are only ever gonna show one in the readme at a time. Maybe change that instead of making another section.